task/testmap.py

```diff
@@ -124,17 +124,17 @@
     'osbuild/cockpit-composer': {
         'master': [
             'fedora-32',
-            'fedora-32/firefox',
             'fedora-33',
+            'fedora-33/firefox',
```

Comment:
This is a new context, so you cannot trigger it in this PR.
It first needs to go into `_manual`, then you can test it and then we can move it to `master`.

Reply:
Make sense to me. Thanks @marusak!